pyvrp/cpp/search/Route.h

```diff
@@ -79,13 +80,17 @@ class Route
 private:
     struct NodeStats
     {
-        Distance cumDist;             // Cumulative dist to this node (incl.)
-        Load cumLoad;                 // Cumulative load to this node (incl.)
+        Distance cumDist;  // Cumulative dist to this node (incl.)
+
+        LoadSegment ls;        // Node's load data
+        LoadSegment lsAfter;   // LS of client -> depot (incl)
+        LoadSegment lsBefore;  // LS of depot -> client (incl)
+
         TimeWindowSegment tws;        // Node's time window data
         TimeWindowSegment twsAfter;   // TWS of client -> depot (incl.)
         TimeWindowSegment twsBefore;  // TWS of depot -> client (incl.)
 
-        NodeStats(TimeWindowSegment const &tws);
+        NodeStats(LoadSegment const &ls, TimeWindowSegment const &tws);
     };
```

Comment:
These should probably become separate vectors by now, rather than one vector of this struct.

Reply:
See #476 for an issue that tracks this.